scripts/export_event_types.py

```diff
@@ -152,6 +160,30 @@
                     )
                 )
 
+        if plot_event_types:
+            Path("plots").mkdir(parents=True, exist_ok=True)
+            # Plot and save the event type distributions.
+            if true_event_types:
+                distribution = event_types.plot_event_type_distribution(
+                    [dtf["event_type"], dtf_7["event_type"], dtf["true_event_type"], dtf_7["true_event_type"]],
+                    label=["all", "gamma-like", "all true", "gamma-like true"],
+                    n_types=n_types,
+                )
+            else:
+                distribution = event_types.plot_event_type_distribution(
+                    [dtf["event_type"], dtf_7["event_type"]], label=["all", "gamma-like"], n_types=n_types
+                )
+
+            if particle is gamma:
```

Comment:
These `if`s are not very elegant, but it's not really important so you can keep as is.

Reply:
At first we expected that too, but yes, it's not the case. A proton that is classified as gamma-like will still have a very bad direction reconstruction, because all the reconstruction models are made exclusively for gammas. So even if a proton has a low reconstructed misdirection, and therefore should go to event type 1, the true misdirection is always huge and it will have a true event type 3. This was producing weird results when computing the sensitivity for true event types, because true event type 1 had almost no background and the sensitivity was unrealistically good.

Reply:
Oops, I answered in the wrong place and I didn't see Tarek had already answered, sorry.